Antidote/ActiveCallViewController.m

```diff
@@ -64,6 +141,28 @@ - (void)installConstraints
         make.bottom.equalTo(self.view.bottom).with.offset(-kIndent);
         make.left.equalTo(self.view.left).with.offset(kIndent);
         make.right.equalTo(self.view.right).with.offset(-kIndent);
+        make.height.equalTo(kEndCallButtonHeight);
+    }];
+
+    [self.videoButton makeConstraints:^(MASConstraintMaker *make) {
+        make.bottom.equalTo(self.view.centerY).with.offset(-kIndent);
+        make.centerX.equalTo(self.view.centerX);
+        make.width.equalTo(kButtonSide);
+        make.height.equalTo(kButtonSide);
+    }];
+
+    [self.microphoneButton makeConstraints:^(MASConstraintMaker *make) {
+        make.top.equalTo(self.videoButton.bottom).with.offset((1.0 / 6.0) * self.view.center.x);
+        make.centerX.equalTo(self.view).with.offset(-(1.0/ 3.0) * self.view.center.x);
```

Comment:
Instead of this it would be better to 
- put microphone and mute button into invisible resizable container
- set gap between views
- center container

So for buttons it would be something like that
```
// microphoneButton
    make.left.equalTo(container.left);
    make.width.equalTo(kButtonSide);
    make.height.equalTo(kButtonSide);

// muteButton
    make.left.equalTo(microphoneButton.right);
    make.right.equalTo(container.right);
    make.width.equalTo(kButtonSide);
    make.height.equalTo(kButtonSide);

// container
    make.centerX.equalTo(self.view)
    ...
```